.github/workflows/release_file.yml

```diff
@@ -0,0 +1,28 @@
+name: Release File
+
+on: 
+  release:
+    types: [published]
+  workflow_dispatch:
+
+jobs:
+  upload_archive:
+    name: Upload archive
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          ref=${{ github.ref_name }}
+          echo "release_name=manifold-${ref:1}" >> $GITHUB_ENV
+      - uses: actions/checkout@v4
+        with:
+          path: ${{ env.release_name }}
+          submodules: recursive
+      - name: Build archive
+        run: >
+          tar  --exclude=".git*" -cz ${{ env.release_name }} -f ${{ env.release_name }}.tar.gz
+      - name: Add file to release
+        run: |
+            cd ${{ env.release_name }} || die
+            gh release upload ${{ github.ref_name }} ../${{ env.release_name }}.tar.gz
```

Comment:
Does this override whatever Github's default release packaging is? Looks great, though I don't have much ability to review or test this. Should we just see what happens on the next release?

Reply:
Sounds good, I'll trust you. Looks like the only problem is our CI - it started misbehaving yesterday for no apparent reason. I'm hoping M$ fixes it soon, though if you have any ideas what could be randomly canceling our jobs or how to work around it, I'd love to know! I already tried a few things to no avail: #826.

Reply:
Looks like the CI started working again - can you prod this PR with a minor commit to trigger the CI? My manual triggering doesn't rerun the randomly cancelled jobs.

Reply:
No problem - in general though I prefer no rebasing and no force-pushing. We do squash and merge anyway, so there's no history pollution. Merging makes things much easier to review because it keeps commits separate. Plus it's safer.